ibis/backends/pandas/udf.py

```diff
@@ -249,7 +250,21 @@ def execute_udaf_node_groupby(op, *args, aggcontext, **kwargs):
             def aggregator(first, *rest):
                 # map(next, *rest) gets the inputs for the next group
                 # TODO: might be inefficient to do this on every call
-                return func(first, *map(next, rest))
+                result = func(first, *map(next, rest))
```

Comment:
same

Reply:
This I didn't change because this is inner loop and a bit too messy to make `coerce_to_output` to work with this

Reply:
ok can you create an issue to fix this up (as your comment indicates)

Reply:
Sounds good! Added https://github.com/ibis-project/ibis/issues/2496